testing/scenario_app/ios/Scenarios/ScenariosUITests/TextSemanticsFocusTest.m

```diff
@@ -45,7 +45,7 @@ - (void)setUp {
   [self.application launch];
 }
 
-- (void)testAccessibilityFocusOnTextSemanticsProducesCorrectIosViews {
+- (void)skip_testAccessibilityFocusOnTextSemanticsProducesCorrectIosViews {
```

Comment:
I've been slapped on the wrist for skipping tests instead of deleting them... I'm cool with it though.